crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -286,13 +292,33 @@ It can greatly improve performance (and readability) in such situations as it pr
 
         let timeline_name = timeline.name().to_string();
 
-        let time_series = system_output.view_systems.get::<TimeSeriesSystem>()?;
+        let legacy_time_series = system_output.view_systems.get::<LegacyTimeSeriesSystem>()?;
+        let line_series = system_output.view_systems.get::<SeriesLineSystem>()?;
+        let point_series = system_output.view_systems.get::<SeriesPointSystem>()?;
 
-        let aggregator = time_series.aggregator;
-        let aggregation_factor = time_series.aggregation_factor;
+        let all_lines: Vec<_> = legacy_time_series
```

Comment:
```suggestion
        let all_plot_items: Vec<_> = legacy_time_series
```

It's not just lines!